FiveCalls/FiveCalls/EditLocationViewController.swift

```diff
@@ -29,14 +38,15 @@ class EditLocationViewController : UIViewController {
     }
     
     @IBAction func useMyLocationTapped(_ sender: Any) {
-        
+        locationManager.requestWhenInUseAuthorization()
     }
     
     @IBAction func cancelTapped(_ sender: Any) {
         delegate?.editLocationViewControllerDidCancel(self)
     }
     
     @IBAction func submitZipCodeTapped(_ sender: Any) {
+        //TODO: delete location if present
```

Comment:
Does this comment still make sense if we are deleting the defaults on updateWith below? Or did you mean in the controller itself?